### ISSUE TITLE
Implement REST Endpoint: Login

### ISSUE THREAD

**Issue by sdonovan1985** [atlanticwave-sdx/atlanticwave-proto#46]
With the overhaul, we need to handle login/logout/etc. 

It is currently handled on the web interface, but I'm not clear how it should work on the REST API.

**Comment by JoaquinChung** [comment]
What about configuration tokens like Corsa does?

**Comment by sdonovan1985** [comment]
Ok... this is a bit more complicated than I thought.

https://flask-login.readthedocs.io/en/latest/#your-user-class - Need a user class. Right now there's only one user, so need to update the UserManager, have a REST API user class. How to do this properly? Should there be another DB that links the actual users and the REST User class? What's the best way?

Lots of tutorials (there are a bunch) integrate with SQLAlchemy, have their own internal Users... Interesting, but not what we're looking for.

So.... What's next? 

Need to try out a few things based on what I've been reading. 
 - Implement UserManager
 - Implement user for Flask
 - Implement new login function
 - Implement new cookie function

**Comment by sdonovan1985** [comment]
Calling this one complete. Reviewed code, did a bunch of testing, and it seems solid (for now). 

Of this list:
 - Implement UserManager
 - Implement user for Flask
 - Implement new login function
 - Implement new cookie function

 The first was covered in #57. The second was more-or-less correct as before, but we did have to implement a flask_login.user_loader function. The third also included a logout function. I did not have to do the last at all, as I believe that this was more of a misunderstanding than anything else.
